site/src/components/TemplateVersionEditor/TemplateVersionEditor.tsx

```diff
@@ -217,13 +225,6 @@ export const TemplateVersionEditor: FC<TemplateVersionEditorProps> = ({
 
             <Button
               variant="contained"
-              title={
-                dirty
-                  ? "You have edited files! Run another build before updating."
-                  : templateVersion.job.status !== "succeeded"
-                  ? "Something"
-                  : ""
-              }
```

Comment:
I removed this because browsers disable `title` whenever the element they apply to is `disabled`. The only text that ever had a chance of being seen by a user was the `"Something"` placeholder text. 😅